tcl/helpbrowser.tcl

```diff
@@ -1,5 +1,5 @@
 
-package provide helpbrowser 0.2
+package provide helpbrowser 0.3
```

Comment:
The version needs to be bumped in pkgIndex.tcl as well.

Reply:
oh yeah I should've checked